radixdlt/src/main/java/com/radixdlt/consensus/validators/ValidatorSet.java

```diff
@@ -27,21 +28,35 @@
 import com.radixdlt.crypto.Hash;
 
 /**
- * Set of validators for consensus.
+ * Set of validators for consensus. Only validators with power >= 1 will
+ * be part of the set.
  * <p>
  * Note that this set will validate for set sizes less than 4,
  * as long as all validators sign.
  */
 public final class ValidatorSet {
 	private final ImmutableBiMap<ECPublicKey, Validator> validators;
 
+	// We assume that we won't have more than 2^128 validators (2^32 is in fact the limit)
```

Comment:
I think this comment is not quite right anymore.  We assume that there will be less than 2^256 staking tokens.